### ISSUE TITLE
DAR000: Unexpected exception in darglint version 1.2.1

### ISSUE THREAD

**Issue by sackh** [terrencepreilly/darglint#78]
Darglint was not throwing any exception on my code, but post upgrading to version 1.2.1.
I am getting below exception -
DAR000: Unexpected exception in darglint: 'Attribute' object has no attribute 'id'.
Also line no I am getting for all these errors is 1.
Is this a valid error?

**Comment by vbuaraujo** [comment]
I'm having this problem with `except` clauses listing multiple exceptions. I can reproduce this error with the following snippet:

```python
def foo():
    """
    Does nothing.

    Raises:
        ValueError: when the world is over.
    """
    try:
        pass
    except (ValueError, KeyError):
        pass
```

**Comment by sobolevn** [comment]
The same happens to me:

```
Linting path: .
flake8 --version:
3.7.9 (flake8-bandit: 2.1.2, flake8-broken-line: 0.2.0, flake8-bugbear: 19.8.0, flake8-comprehensions: 3.2.2, flake8-darglint: 0.4.1, flake8-debugger: 3.2.1, flake8-docstrings: 1.5.0, pydocstyle: 5.0.2, flake8-eradicate: 0.3.0, flake8-string-format: 0.2.3, flake8_commas: 2.0.0, flake8_isort: 2.9.1, flake8_quotes: 2.1.2, mccabe: 0.6.1, naming: 0.9.1, pycodestyle: 2.5.0, pyflakes: 2.1.1, rst-docstrings: 0.0.12, wemake_python_styleguide: 0.14.0) CPython 3.7.7 on Linux
=================================

_get_version: 'Attribute' object has no attribute 'id'
test_no_exceptions: 'Tuple' object has no attribute 'id'
./tests/test_checker/test_hypothesis.py:1:1: DAR000: Unexpected exception in darglint: 'Tuple' object has no attribute 'id'
=================================

./wemake_python_styleguide/version.py:1:1: DAR000: Unexpected exception in darglint: 'Attribute' object has no attribute 'id'
./tests/test_checker/test_hypothesis.py:1:1: DAR000: Unexpected exception in darglint: 'Tuple' object has no attribute 'id'
Process failed with the status code: 1
```

Reproduction url: https://github.com/wemake-services/wemake-python-styleguide/pull/1311/checks?check_run_id=563057590

**Comment by terrencepreilly** [comment]
Yes, I noticed this this morning as well.  It only happened for me with Python3.8: 3.5 worked fine.  I'll take a closer look as soon as I get a chance.

**Comment by jmaroeder** [comment]
Also seems to be happening with a single, dotted exception, as in the following:

```python
import requests


def foo():
    try:
        pass
    except requests.exceptions.RequestException:
        pass
```
```console
$ flake8 simple_case.py
foo: 'Attribute' object has no attribute 'id'
```

**Comment by sobolevn** [comment]
Looks like it is fixed in `1.2.2` and can be closed.
Thanks a lot!
